pkgs/top-level/all-packages.nix

```diff
@@ -7718,6 +7718,8 @@ with pkgs;
 
   modsecurity_standalone = callPackage ../tools/security/modsecurity { };
 
+  modsecurity-csr = callPackage ../tools/security/modsecurity-csr { };
```

Comment:
Typo in both the attr and the path. It's `crs`, not `csr`

Reply:
Oops, fixed. Thanks.